packages/html/ds/src/textarea/textarea.schema.ts

```diff
@@ -0,0 +1,33 @@
+import * as zod from 'zod';
+
+export const textAreaSchema = zod.object({
```

Comment:
Not sure I understand the schema. I can't see the `hintText`, `errorText` or `label` prop. Are we purposely leaving them out ?